src/main/java/com/example/onlinebookstore/service/impl/UserServiceImpl.java

```diff
@@ -39,7 +39,6 @@ public UserResponseDto save(
                 () -> new IllegalStateException("Default role not found"));
         user.setRoles(Collections.singleton(defaultRole));
         User savedUser = userRepository.save(user);
-        shoppingCartService.createForUser(savedUser);
```

Comment:
why remove it?

Reply:
I think accidentally while resolving conflicts